app/presenters/contact_presenter.rb

```diff
@@ -48,7 +48,6 @@ def contact_details
       slug: contact.slug,
       title: contact.title,
       description: contact.description,
-      organisation: ContactOrganisationPresenter.new(contact.organisation).present,
```

Comment:
I wondered if removing this meant we could also remove the `ContactOrganisationPresenter` class itself.  Turns out it's used by the `ContactGroupPresenter` to embed the organisation details.  The `ContactGroupPresenter` in turn is used to present all the groups that the contact is a member of within the `details` hash of the `ContactPresenter`.  So at the moment, no we can't drop it.  But we should probably make a note to think about it and drop it later.

As another aside we might want to think about whether or not we should validate that a `Contact` isn't added to a `ContactGroup` that belongs to a different organisation.  Slightly moot at the moment as the only live org that uses contacts-admin is HMRC, but that may not always be the case.